### ISSUE TITLE
Grammar feedback

### ISSUE THREAD

**Issue by jbrugge** [w3c/publ-a11y#486]
This is a set of suggestions that I am grouping as one issue because each are small. I list the section each is in so that the exact spot of the phrase is as clear as possible.

1. Abstract: remove "proposes" from "These guidelines document proposes a...".
2. Introduction: remove ", and more" from "All of these, and more...".
3. Introduction: change "after reading a paragraph" to "before reading a paragraph". The context of the sentence suggests that the goal is to alert a reader before encountering something rather than after.
4. Terminology: for electronic braille, change ", which denotes" to ", and denote".
5. General overview: "important accessibility claims that helps", the word "help" should be singular.
6. General overview: "is meant to be machine readable - except", use a comma rather than a dash.
7. General overview: "- in this way", change the dash to a period and begin a new sentence.
8. General overview: "in a slightly different way, some", change the comma to a period and begin a new sentence.
9. Metadata processing overview: in the second "Note", change the parenthetical example from "(e.g. ...)" to a sentence, "For example, if a..."
10. Metadata processing overview: clarify the example, which feels incomplete. I might interpret it as follows, but it is unclear if this is what was meant: "For example, if a vendor prefers to handle only ONIX metadata for display but only receives EPUB or PDF, then they would need to preprocess the metadata embedded in an EPUB or PDF to create ONIX."
11. Key accessibility information: third bullet point, drop the comma in "such as libraries or schools, need to".
12. Localization strategy: "local actors like NGOs libraries serving" seems like it could be either NGOs or libraries, or perhaps "NGO libraries", the meaning isn't clear. Consider dropping either "NGOs" or "libraries".
13. Localization strategy: "Reversely" doesn't not sound like a valid word. Perhaps the meaning could be captured by "As well,...".
14. Localization strategy: the sentence "And has already been used" is part of the previous sentence and can be connected, as "in early 2022, and has already been used."
